CMakeLists.txt

```diff
@@ -42,6 +42,21 @@ set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Build Halide with ccache if the package is present
+set(Halide_CCACHE_BUILD OFF CACHE BOOL "Set to ON for a ccache enabled build")
```

Comment:
```
option(Halide_CCACHE_BUILD "Set to ON for a ccache enabled build" OFF)
```
I prefer `option()` for easier grepping and because normal variables can override it (eg. in a build that includes us).

Reply:
Done